src/gui/accountsettings.cpp

```diff
@@ -444,13 +479,18 @@ void AccountSettings::slotSubfolderContextMenuRequested(const QModelIndex& index
     auto acc = _accountState->account();
 
     if (acc->capabilities().clientSideEncryptionAvaliable()) {
+        // Verify if the folder is empty before attempting to encrypt.
+        qDebug() << "info->path" << info->_path;
+        qDebug() << info->_folder->path();
+        qDebug() << info->_folder->cleanPath();
+
```

Comment:
If you want to keep the qDebug maybe you could change it to qCDebug/info/warning?

Reply:
don't want to keep it, updating.